### ISSUE TITLE
Support Imgbox

### ISSUE THREAD

**Issue by Dyras** [ShareX/ShareX#2981]
Greetings! Imgbox has been around since 2010 and they don't compress images at all. I think it would be a neat addition to ShareX.

**Comment by damian5000** [comment]
I agree. Was just looking at ImgBox since Imgur has banned Thai uploads, but don't see ShareX support for ImgBox.

**Comment by Dyras** [comment]
@Jaex Any possibility of adding this? 🙏

**Comment by sergeevabc** [comment]
+1

**Comment by Speedy2662** [comment]
+1
